### PR TITLE
Google AdWords documentation update

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Updated AdWords documentation with updated information on adding custom reports as a resource.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by mcchran — APPROVED** [review]
LGTM.

File: cloud-extract-sources/google-adwords.mdx [hunk 1]
```diff
@@ -6,141 +6,110 @@ description: Step-by-step guide to ingest your Google AdWords data into RudderSt
...
+| Campaigns                     | `campaigns`                    | Contains information about the campaigns in your AdWords account. The columns in this table are `name`, `startDate`, and `frequencyCap` with `id` as the primary key.  |
+| Report: Keywords Performance  | `keywords_performance_report`  | Contains information about your AdWords keywords' performance. The columns in this table are `ActiveViewImpressions`, `AverageCost`, `AverageCpc`, `CampaignName`, and `KeywordMatchType`.    |
+| Report: Click Performance     | `click_performance_report`     | Contains information about your AdWords' click performance. The columns in this table are `AccountDescriptiveName`, `AdGroupStatus`, `CampaignId`, `Clicks`, and `ClickType`.                           |
+| Report: Placement Performance | `placement_performance_report` | Contains the your AdWords account's placement performance. The columns in this table are `AverageCpc`, `AdGroupStatus`, `Clicks`, `Device`, and `Ctr`.                                         |
+
+## FAQs
```
Comment (ishachopra0401) [inline cloud-extract-sources/google-adwords.mdx:107]:
```suggestion
## FAQ
```

File: cloud-extract-sources/google-adwords.mdx [hunk 2]
```diff
@@ -6,141 +6,110 @@ description: Step-by-step guide to ingest your Google AdWords data into RudderSt
...
+### Source settings
 
-<!--<img src="../assets/screen-shot-2020-12-17-at-4.39.38-pm.png" />-->
+- **Customer ID**: After you connect RudderStack to your Google AdWords account, the associated AdWords customer IDs will automatically appear in this dropdown. 
 
-<img
-  src="../assets/screen-shot-2020-12-17-at-4.39.38-pm.png"
-  alt="Connect with Google AdWords"
-/>
+- **Start date**: This option specifies the earliest date when RudderStack will import all your historical AdWords data.
```
Comment (ishachopra0401) [inline cloud-extract-sources/google-adwords.mdx:39]:
```suggestion
- **Start date**: This option specifies the earliest date by when RudderStack can import all your historical AdWords data.
```

File: cloud-extract-sources/google-adwords.mdx [hunk 3]
```diff
@@ -6,141 +6,110 @@ description: Step-by-step guide to ingest your Google AdWords data into RudderSt
...
+<img src="../assets/cloud-extract-sources/google-adwords-source-settings.png" alt="AdWords source settings" />
 
-<!--<img src="../assets/screen-shot-2020-12-17-at-4.40.01-pm.png" />-->
+### Destination settings and data update schedule
 
-<img
-  src="../assets/screen-shot-2020-12-17-at-4.40.01-pm.png"
-  alt="Customer ID"
-/>
+- **Table prefix**: RudderStack will create a table in your data warehouse with this prefix name and load all your AdWords data into it.
```
Comment (ishachopra0401) [inline cloud-extract-sources/google-adwords.mdx:52]:
```suggestion
- **Table prefix**: Creates a table in your data warehouse with the specified prefix name and loads all your AdWords data into it.
```

File: cloud-extract-sources/google-adwords.mdx [hunk 4]
```diff
@@ -6,141 +6,110 @@ description: Step-by-step guide to ingest your Google AdWords data into RudderSt
...
+<img src="../assets/cloud-extract-sources/google-adwords-resources.png" alt="AdWords resources to import" />
 
-<img
-  src="../assets/screen-shot-2020-12-17-at-4.40.48-pm.png"
-  alt="Run Frequency"
-/>
+- You can also import a custom report as a resource. Refer to the [Adding a custom report as resource](#adding-custom-reports-as-resources) section below for more information.
 
-### Specifying the Data to Sync
+That's it! Google AdWords is now successfully configured as a source. RudderStack will start importing the Google AdWords data as per the specified frequency.
```
Comment (ishachopra0401) [inline cloud-extract-sources/google-adwords.mdx:68]:
```suggestion
Google AdWords is now successfully configured as a source. RudderStack will start importing the Google AdWords data as per the specified frequency. 
```

File: cloud-extract-sources/google-adwords.mdx [hunk 5]
```diff
@@ -6,141 +6,110 @@ description: Step-by-step guide to ingest your Google AdWords data into RudderSt
...
-  src="../assets/screen-shot-2020-12-17-at-4.40.48-pm.png"
-  alt="Run Frequency"
-/>
+- You can also import a custom report as a resource. Refer to the [Adding a custom report as resource](#adding-custom-reports-as-resources) section below for more information.
 
-### Specifying the Data to Sync
+That's it! Google AdWords is now successfully configured as a source. RudderStack will start importing the Google AdWords data as per the specified frequency. 
 
-- Next, you need to specify the data you want RudderStack to import. By default, all the data resources are selected.
+You can further connect this source to your data warehouse by clicking on **Connect Destinations** or **Add Destinations**, as shown:
```
Comment (ishachopra0401) [inline cloud-extract-sources/google-adwords.mdx:70]:
```suggestion
You can further connect this source to your data warehouse by clicking on **Add Destination**, as shown:
```

File: cloud-extract-sources/google-adwords.mdx [hunk 6]
```diff
@@ -6,141 +6,110 @@ description: Step-by-step guide to ingest your Google AdWords data into RudderSt
...
+<img src="../assets/cloud-extract-sources/google-adwords-connect-destinations.png" alt="Connecting a destination to AdWords source" />
 
-<img
-  src="../assets/screen-shot-2020-12-17-at-4.41.10-pm.png"
-  alt="Specifying the data to sync"
-/>
+<div class="successBlock">
 
-That's it! Google AdWords is now successfully configured as a source on your RudderStack dashboard.
+  Use the <strong>Connect Destinations</strong> option if you have already configured a data warehouse destination in RudderStack/ To configure a warehouse destination from scratch, click on the <strong>Add Destination</strong> button.
```
Comment (ishachopra0401) [inline cloud-extract-sources/google-adwords.mdx:76]: Can't locate 'Connect Destinations' option.

**Review by ishachopra0401 — CHANGES_REQUESTED** [review]

File: cloud-extract-sources/google-adwords.mdx (hunk 6 above)
Comment (ameypv-rudder) [reply cloud-extract-sources/google-adwords.mdx]: fixed this.

**Review by ishachopra0401 — APPROVED** [review]

**Pull request merged**